code/modules/multiz/mz_items.dm

```diff
@@ -0,0 +1,55 @@
+/obj/item/weapon/ladder_mobile
+	name = "mobile ladder"
+	desc = "A lightweight deployable ladder, which you can use to move up or down. Or alternatively, you can bash some faces in."
+	icon_state = "ladder01"
+	icon = 'icons/obj/structures.dmi'
+	throw_range = 3
+	force = 10
+
+/obj/item/weapon/ladder_mobile/proc/place_ladder(atom/A,mob/user)
+
+	if(istype(A, /turf/simulated/open))         //Place into open space
+		user.visible_message("<span class='warning'>[user] begins to lower the ladder into the hole.</span>")
```

Comment:
Needs a second argument: that's being shown to the `user`. Something like, `"<span class='warning'>You begin to lower \the [src] into \the [A].</span>"`. It might also be an idea to move this to be right before handle_action. Otherwise, if below_loc == space, then you get _this message_ along with the failure message. Which is weird.